CHANGES.md

```diff
@@ -1,3 +1,9 @@
+# UNRELEASED
+
+- Runtime/FFI:
+  - fix handling of eta expanded unification variables. Many thanks to
+    @nguermond for testing this tricky case.
```

Comment:
@nguermond I can put your name if you prefer.

Reply:
It's Nathan Guermond, thanks!